apps/passport-client/components/screens/LoginScreens/AlreadyRegisteredScreen.tsx

```diff
@@ -41,6 +41,7 @@ export function AlreadyRegisteredScreen() {
   const identityCommitment = query?.get("identityCommitment");
   const [error, setError] = useState<string | undefined>();
   const [isLoggingIn, setIsLoggingIn] = useState(false);
+  const [isLoadingUser, setIsLoadingUser] = useState(false);
```

Comment:
can we combine this with the `isLoggingIn` state since that was supposed to be what solved this issue in the first place, and it doesn't look like we need two separate states for rendering anything.

Reply:
if i just remove all instances of isLoadingUser and then only put the setIsLoggingIn(false) within the "if (!storageResult.success)` block, it also seems to fix the flash

Reply:
> if i just remove all instances of isLoadingUser and then only put the setIsLoggingIn(false) within the "if (!storageResult.success)` block, it also seems to fix the flash

nice